### PR TITLE
[rllib] Remove "Common", cleanup some code

### PR DESCRIPTION
**Pull request by richardliaw**
Removed "Common" from "CommonPolicyEvaluator".
 - [x] Waiting for support * PR #2346 to land first

### CODE REVIEW

File: python/ray/rllib/__init__.py [hunk 1]
```diff
@@ -27,6 +35,24 @@ def _register_all():
...
+    "ActionDistribution", "ActionDistribution", "Categorical",
+    "DiagGaussian", "Deterministic", "ModelCatalog", "Model",
+    "Preprocessor", "FullyConnectedNetwork", "LSTM",
+
+    # rllib.optimizers
+    "PolicyOptimizer", "AsyncSamplesOptimizer", "AsyncGradientsOptimizer",
+    "SyncSamplesOptimizer", "SyncReplayOptimizer", "LocalMultiGPUOptimizer",
+
+    # rllib.utils
+    "Filter", "FilterManager", "PolicyClient", "PolicyServer"
```
Comment (ericl) [inline python/ray/rllib/__init__.py]: Hmm this is a lot more than seems necessary, why not stick to the current set of imports and add just a couple more if needed?

**Review by ericl — APPROVED** [review]

File: python/ray/rllib/__init__.py [hunk 2]
```diff
@@ -27,6 +32,22 @@ def _register_all():
...
+    "AsyncVectorEnv", "MultiAgentEnv", "ServingEnv", "VectorEnv", "EnvContext",
+
+    # rllib.evaluation
+    "PolicyEvaluator", "PolicyGraph", "TFPolicyGraph",
+    "TorchPolicyGraph", "compute_advantages", "collect_metrics",
+
+    # rllib.models
+    "ActionDistribution", "ActionDistribution", "Categorical",
+    "DiagGaussian", "Deterministic", "ModelCatalog", "Model",
+    "Preprocessor", "FullyConnectedNetwork", "LSTM",
```
[the remark this answers is not in the document]
Comment (richardliaw) [reply python/ray/rllib/__init__.py]: I think you commented on an old diff... there are no changes to `__init__.py`

**Comment by ericl** [comment]
You'll also have to fix links in the rst files.

**Comment by richardliaw** [comment]
jenkins retest this please

**Comment by richardliaw** [comment]
Test fails at `multi_node_tests`.

**Pull request merged**